### PR TITLE
Update/pm filter sort default

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
😺
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2483)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/769)

### CODE REVIEW

File: src/services/common.js [hunk 1]
```diff
@@ -449,7 +449,9 @@ const bidNameMapping = (val, toWS=false) => {
 const convertTemplateFiltersCols = (query, mapFunc) => {
   const queryFilterDict = {
     EQ: "=",
-    IN: "="
+    IN: "=",
+    GTEQ: ">",
+    LTEQ: "<",
```
Comment (elizabeth-jimenez) [inline src/services/common.js:454]: 😻

File: src/services/agendas.js [hunk 2]
```diff
@@ -428,23 +435,28 @@ const getPanels = async (filsCols, query) => {
...
+            'pmsdesctext': a.pmsdesctext,
+            'pmddttm': panelMeetingDatesdate,
+            'mdtcode': panelMeetingDatesMdtCode,
+            'panelMeetingDates': panelMeetingDatesData,
+          }
         }
       });
 
-      return data;
+      data = _.filter(data, function(o) { return o !== undefined; });
```
Comment (scott062) [inline src/services/agendas.js:457]: Just learned this - you can do this more elegantly where your .map is by using .flatMap and then you don't have to .filter after

Comment (scott062) [reply src/services/agendas.js:457]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#for_adding_and_removing_items_during_a_map

Comment (elizabeth-jimenez) [reply src/services/agendas.js:459]: very noice! thank you, updated!

File: src/services/agendas.js [hunk 3]
```diff
@@ -420,7 +420,7 @@ const getPanels = async (filsCols, query) => {
       data = data.serialize();
       let panelMeetingDatesMdtCode = ''
       let panelMeetingDatesdate = ''
-      data = _.flatMap(data, function(a) {
```
Comment (scott062) [inline src/services/agendas.js]: 🥲

File: Dockerfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Check out https://hub.docker.com/_/node to select a new base image
-FROM node:10-slim
+FROM node:11-slim
```
Comment (scott062) [inline Dockerfile:2]: Note to all devs - you'll need to rebuild the container to see this take effect

**Review by scott062 — APPROVED** [review]
🥿 🫓 🥙 🗺️ 1️⃣ 1️⃣ 
Nice job!

**Pull request merged**